themes/classic/_dev/js/cart.js

```diff
@@ -274,4 +280,38 @@ $(document).ready(() => {
   )
 });
 
+const CheckUpdateQuantityOperations = {
+  'switchErrorStat': () => {
+    let $checkoutBtn = $('.checkout a');
+
+    if ($("#notifications article.alert-danger").length
+        || (hasError && 'decreaseProductQuantity' === updateOpertation)) {
+      $checkoutBtn.addClass('disabled');
+    }
+
+    if (hasError && '' !== errorMsg) {
+      let strError = ' <article class="alert alert-danger" role="alert" data-alert="danger"><ul><li>' + errorMsg + '</li></ul></article>';
+      $checkoutBtn.addClass('disabled');
```

Comment:
Is it really disabled ? Or just visually ?

Reply:
I used the same class that is applied to disable the button saved in the BO product page, it actually disabled the button.
`a.btn.disabled{
    pointer-events: none;
}`

Reply:
:+1: